packages/lib-auth/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@zooniverse/auth",
+  "version": "1.0.0",
+  "main": "src/index.js",
```

Comment:
This new package was originally written as a CJS package. Nowadays, it would make more sense to write it as an ES Module.